CONTRIBUTING.md

```diff
@@ -84,6 +84,7 @@ For example, APM Server version `7.15.0` corresponds with the Go APM Agent versi
 
 1. Update `version.go` and `internal/apmversion/version.go`, and then run `make update-modules`
 2. Update [`CHANGELOG.asciidoc`](changelog.asciidoc), by adding a new version heading (`==== 1.x.x - yyyy/MM/dd`) and changing the base tag of the Unreleased comparison URL
+3. Run `make update-modules` to sync the dependencies version
```

Comment:
I'm a bit late, but this is already mentioned at the end of step 1. Perhaps split it into a following line if it's easy to miss.